README.md

```diff
@@ -542,6 +542,7 @@ And there you have it.
   <tr>
     <td align="center"><a href="http://prolabprints.com/"><img src="https://avatars.githubusercontent.com/u/7111699?v=4" width="100px;" alt=""/><br /><sub><b>Ron Dyar</b></sub></a></td>
     <td align="center"><a href="https://github.com/toddpress"><img src="https://avatars.githubusercontent.com/u/2934422?v=4" width="100px;" alt=""/><br /><sub><b>Todd Pressley</b></sub></a></td>
+    <td align="center"><a href="https://github.com/doesnotexist"><img src="https://avatars.githubusercontent.com/u/304697?v=4" width="100px;" alt=""/><br /><sub><b>Omar El-Domeiri</b></sub></a></td>
```

Comment:
Ah, this is a good reminder for me to update the All Contributors. (It's been too long!🤦‍♂️) This section of code is auto-generated, so please remove this change from the PR.

**Instead, once this PR is merged, either:** 
1. ping/nudge/remind/push me to update
2. or take a look at the process in [tasks/all-contributors/README.md](https://github.com/redwoodjs/redwood/blob/main/tasks/all-contributors/README.md) and open a PR